src/platform.js

```diff
@@ -41,12 +41,12 @@ exports = module.exports = {
 	* `true` if the platform has native single-finger [events]{@glossary event}.
 	* @public
 	*/
-	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints),
+	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints || window.navigator.maxTouchPoints),
```

Comment:
Should we actually be checking if `msMaxTouchPoints`/`maxTouchPoints` is exactly 1 here, and >1 for the gesture feature detection below?